contactsummary/js/test/globals.js

```diff
@@ -1,12 +1,15 @@
+/* globals CRM */
+
 (function (CRM) {
-    CRM.vars = {
-        contactsummary: {
-            baseURL: '/base/tools/extensions/civihr/contactsummary'
-        }
-    };
+  CRM.vars = {
+    contactsummary: {
+      baseURL: '/base/tools/extensions/civihr/contactsummary'
+    },
+    leaveAndAbsences: {}
```

Comment:
is this needed at all?

Reply:
unfortunately yes. the shared-settings module of L&A needs to find this variable in the global scope (CRM.vars.leaveAndAbsences)